### PR TITLE
chromium@122.0.6261.56: Fix hash

### LINKED ISSUE #138: chromium@122.0.6261.56: hash check failed

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #138

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### chromium

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
